docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -24,14 +24,15 @@ Configuration
 **Admin Email:** Zendesk admin email.
 
 **API Token:** Zendesk API token. Can be obtained from the Zendesk Support Admin interface.
-Check out [Zendesk documentation](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
-for API Token generation.
+For information about generating the Zendesk API token, see [Zendesk API Token](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-).
 
-**Subdomains:** List of Zendesk Subdomains to read object from.
+**Subdomains:** List of Zendesk [Subdomains](https://support.zendesk.com/hc/en-us/articles/4409381383578-Where-can-I-find-my-Zendesk-subdomain-) to read object from.
 
-**Object to Pull:** Objects to pull from Zendesk API.
+**Object to Pull:** Objects to pull from Zendesk API.If not specified, data is pulled from all objects. Default is blank.
```

Comment:
Missing a space after the period "...API. If"

Reply:
done